src/Uno.UI/UI/Xaml/Controls/ItemsStackPanel/ItemsStackPanel.cs

```diff
@@ -24,9 +31,20 @@ public ItemsStackPanel()
 			{
 				CacheLength = FeatureConfiguration.ListViewBase.DefaultCacheLength.Value;
 			}
+
+#if NETSTANDARD2_0
```

Comment:
Why is it created earlier in wasm?

Reply:
Because ItemsStackPanel is a real panel on WASM, whereas on Android/iOS it's just a facade. (On WASM GetLayouter() is never called.)